Arduino/McLighting/request_handlers.h

```diff
@@ -302,9 +346,14 @@ void getStatusJSON() {
 }
 
 String listModesJSON(void) {
-  const size_t bufferSize = JSON_ARRAY_SIZE(strip.getModeCount()+1) + strip.getModeCount()*JSON_OBJECT_SIZE(2);
+  const size_t bufferSize = JSON_ARRAY_SIZE(strip.getModeCount()+1) + strip.getModeCount()*JSON_OBJECT_SIZE(2) + 1000;
```

Comment:
Latest Arduino JSON version is very aggressive with memory management, it won't send any data just because of buffer length being small. The previous code without those changes would not work correctly.